### DISCUSSION TITLE
Cosmic voids

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#3121] [Ideas]
Can stellarium also contain the locations of the cosmic voids?
See: https://en.wikipedia.org/wiki/List_of_voids

**Comment by alex-w** [comment]
Many voids has no coordinates. We may add visualization of the markers for center of voids as separate plugin, because at least I do not see importance to see voids for most users of Stellarium.

**Reply by gzotti** [reply]
I would just *not* visualize nothings. Our galaxy catalog is not deep enough to begin with, and there are cosmology simulations that have been created exactly for that purpose. The voids become most interesting in a 3D or volumetric representation of billions of lightyears seen "from the outside" and moved/rotated as volumetric object.

**Comment by tzmosh** [comment]
Thank you!
